.editorconfig

```diff
@@ -0,0 +1,14 @@
+root = true
+
+[*]
+charset = utf-8
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.yml]
+indent_size = 2
```

Comment:
Note that [`.travis.yml`](https://github.com/fzaninotto/Faker/blob/1273b5dd71b42cd7e0d9a8551cb0a5675ebc117c/.travis.yml) is currently indented with 2 and 4 spaces. Personally, I'd prefer to use 2 spaces for indentation of `yml` files, as otherwise indentation and wrapping of nested objects is a bit weird.

I'd follow up (or push into this branch if you prefer) an adjusted `.travis.yml` with consistent indentation.

Reply:
Agree, lets reindent it